pandas/core/indexes/datetimelike.py

```diff
@@ -121,6 +121,45 @@ def floor(self, freq):
 
     @Appender(_round_doc % "ceil")
     def ceil(self, freq):
+        """
+        Floor the index to the specified freq.
+
+        Links to the available [frequency strings](http://pandas.
+        pydata.org/pandas-docs/stable/timeseries.html#offset-aliases)
+        and [frequency objects](http://pandas.pydata.org/pandas-docs/
+        stable/timeseries.html#dateoffset-objects).
+
+        Parameters
+        ----------
+        freq : str, object
+            String or object specifying the frequency to round down to.
+
+        Returns
+        -------
+        index of same type
+
+        See Also
+        --------
+        DatetimeIndex.floor : Floor the DatetimeIndex to the specified freq.
+        DatetimeIndex.round : Round the DatetimeIndex to the specified freq.
+
+        Examples
+        --------
+
+        >>> dti = pd.DatetimeIndex(start='2014-08-01 09:23:41.321211',
+        freq = 'H', periods = 3)
```

Comment:
`idx` is more standard than `dti` in the docs. Also, the format should be:
```
>>> dti = pd.DatetimeIndex(start='2014-08-01 09:23:41.321211',
...                        freq='H', periods=3)
```
Note the `...` at the beginning of the second line, its alignment, and that I removed the spaces around `=`, which are not PEP-8 valid.